backend/api/Models/AccessRequest/AccessRequestModel.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections.Generic;
+
+namespace Pims.Api.Models.AccessRequest
+{
+    public class AccessRequestModel : BaseAppModel
+    {
+        #region Properties
+        public long Id { get; set; }
+        public UserModel User { get; set; }
+        public RoleModel Role { get; set; }
+        public IEnumerable<AccessRequestOrganizationModel> Organizations { get; set; }
+        #endregion
```

Comment:
same comment, I think we still want the status

Reply:
The schema doesn't have a status anymore.  Now I guess the logic would be, an access request has been created and is relevant until a user account has been activated and has a role and organization they are associated with.  Of course we don't have any seed data for organizations yet, so it's a bit difficult.